MANIFEST

```diff
@@ -0,0 +1,32 @@
+AUTHORS
+bin/wg-meta
+CHANGES
+COPYRIGHT
+lib/Wireguard/WGmeta.pod
+lib/Wireguard/WGmeta/Cli.pod
```

Comment:
and this into the README.md

there is no need to have a file at every directory level